src/main.ts

```diff
@@ -235,9 +235,13 @@ async function run_linux(): Promise<void> {
 
     for (const pkg of dpkg_diff) {
       const output = await capture(`sudo dpkg -L ${pkg}`, {silent: true})
-      const files: Array<string> = output
-        .split('\n')
-        .filter(f => fs.statSync(f).isFile())
+      const files: Array<string> = output.split('\n').filter((f: string) => {
+        try {
+          return fs.statSync(f).isFile()
+        } catch (e) {
+          return false
+        }
+      })
```

Comment:
Am I understand right that here we catch an exception from `fs.statSync()` regarding the non-existent file? If so, it would be more accurate to pass the `throwIfNoEntry: false` [flag](https://nodejs.org/api/fs.html#fsstatsyncpath-options).

However, silent ignoring of all the non-existent files anyway looks a bit dangerous. I guess I can propose a better way.

See, the root of the problem is in `dpkg` excludes in the docker container.

```sh
$ cat /etc/dpkg/dpkg.cfg.d/excludes 
# Drop all man pages
path-exclude=/usr/share/man/*

# Drop all translations
path-exclude=/usr/share/locale/*/LC_MESSAGES/*.mo

# Drop all documentation ...
path-exclude=/usr/share/doc/*

# ... except copyright files ...
path-include=/usr/share/doc/*/copyright

# ... and Debian changelogs for native & non-native packages
path-include=/usr/share/doc/*/changelog.*
```

`apt-get install cmake` respects them, but `dpkg -L` doesn't. The file is created during build of the docker image, see [here](https://git.launchpad.net/livecd-rootfs/tree/live-build/auto/build?h=ubuntu/jammy#n179).

AFAIS, `dpkg -L` doesn't respect to explicitly passed `--exclude-path` and `--include-path` options. But we can just implement this filtering in Typescript.

I would hardcode the exclude/include rules as in the file listed above in the action code.

I don't insist, here are just my thoughts around.

However, I would describe the reason of the problem a bit deeper in the commit message (like it is done in this message, for example).

Reply:
I would also add that it is more related to run in a docker container job rather than Ubuntu of a particular version. I guess that it is as related to Ubuntu 22.04 as for 20.04.